### ISSUE TITLE
look at sub-zero chill papers and related from OSPREE....

### ISSUE THREAD

**Issue by lizzieinvancouver** [lizzieinvancouver/chillingconcepts#7]
From OSPREE getchill.R has some good stuff we may want to check for our chilling review paper

`"cook05"     "schnabel87"` have freeze treatments!

The below appear to have sub-zero chilling treatments: 

```
> unique(chillneg$datasetID) # 7 studies, incl. jones12, lamb37, granhus09
[1] "granhus09"  "heide15"    "jones12"    "lamb37"     "man10"      "pop2000"   
[7] "sonsteby14"
```

And, if you have time, the below may have chilling with photoperiods, but I did not check which actually VARY photoperiod during chilling, those seem the only ones worth looking at.

```
> interestingchillphotoperiods <- c("10", "9", "8", "16", "12", "13", "AB")
> chillphoto <- bb[which(bb$chillphotoperiod %in% interestingchillphotoperiods),]
> unique(chillphoto$datasetID) # 11 incl. worrall67, man17, nienstaedt66
 [1] "boyer"        "caffarra11b"  "campbell75"   "Heide03"      "jones12"     
 [6] "li05"         "man17"        "myking95"     "nienstaedt66" "spiers74"    
[11] "worrall67"  
```

**Comment by lizzieinvancouver** [comment]
If you want to investigate any of these in R further, before you look at the papers, the code [is here](https://github.com/lizzieinvancouver/ospree/blob/master/analyses/misc/getchill.R).

**Comment by ngoj1** [comment]
**Papers with explicit freeze treatments:**
 - cook05: _Malus domestica_ 'Granny Smith' / Compared to chilling treatments of 1, 4, 7, or 10 degC (which, between the 4, had no significant difference in efficiency for budburst) the -1/13 degC night/day treatment displayed delayed bud growth and also prolonged dormancy / a later experiment in 2003 showed that chilling at 1 degC was much more efficient than 5, 9, or 13 degC for budburst
- schnabel87: _Vitis vinifera_ / This paper tested for cold hardiness, not budburst
 
 **Papers with subzero chilling treatments:**
- sonsteby14: _Ribes nigrum_ / optimal bud burst at 2 degC for 6 or more weeks / Figure 3 and 5 show that temps of 0 or -5 degC tended to have the highest percentage of budbreak and also the lowest number of days to bud break compared to 5, 10, and -10 degC
 - pop2000: _Salix pulchra_ & _Betula nana_ / didn't detect any experiment involving subzero, but it's mentioned in their usage of Hanninen's chilling units model
 - man10: _Picea glauca_ / chilling of 0/10 degC night/day had the most delayed bud burst progression / mean daily temperature of 10/20 degC night/day had fewest days to burst / second-best thermal regime for budburst was 0/10 day/night with a flat increase for both temps by 3 degC every 6 days / suggests that freezing temp delayed budburst (my own assumption: ecodormancy mechanism?)
 - lamb37 (technically its 1948?): _Rubus idaeus_ 'Latham' / dormant canes held at 3 degC took around half as long to burst compared to those held at -3 degC across the duration of the experiment, suggesting that the chill requirement is twice as fast when held at the optimal temperature / canes exposed to alternating 3/-3 degC 16/8 hr thermoperiod took the longest to burst, which supports the idea of prolonged chilling below a certain temperature to accumulate something (maybe it's like that model with the intermediate molecule)
 - jones12: _Ribes nigrum_ cultivars / shoots chilled at -5 degC had higher percentages overall of achieved bud stages (i.e. more had reached bud stage 1) compared to those chilled at 5 degC after exposure to 20 degC permissive temperature / in general, plants exposed to -5 degC chilling had the highest percentages of plants that had achieved a certain bud stage
 - heide15: _Ribes nigrum_ / plants were just chilled at -1 degC for 10 weeks to break dormancy, no change in temperature tested
 - granhus09: _Picea abies_ / 0 degC (0.7 +- 0.7 degC) was the chilling treatment after a dormancy induction treatment of either 12 or 21 degC / days to bud burst decreased with increasing chilling time / days to bud burst increased when the dormancy induction temp was higher (21 degC)

**Those other papers where photoperiod and chill might have been wack**
 - caffarra11: _Betula pubescens_ / exp3: in chilled plants a lower dormancy induction temp (10 degC vs 18 degC) both decreased time to budburst and also increased budburst % ; photoperiod during chilling (10 degC with either 8hr or 0hr) had no effect on budburst % or occurrence / exp4: constant short photoperiod led to higher % budburst and shorter budburst time during dormancy induction compared to plants in a treatment where photoperiod naturally decreased, but like exp3 photoperiod during chilling did not matter
 - boyer: _Pinus taeda_ / longer photoperiods prior to chilling delayed budburst / Time to budbreak was generally shortest at increasing chill duration and decreasing photoperiod (Table 2)
 - campbell75: _Pseudotsuga menziesii_ / the experiment that tests photoperiod changes had universal chill temp of 4 degC
 - heide03: _Betula pendula_, _Betula pubescens_, _Alnus glutinosa_ / Tested temperature during dormancy induction, not necessarily photoperiod
 - jones12: _Ribes nigrum_ cultivars / didn't manipulate photoperiod
 - li05: _Betula pendula_ / didn't change the photoperiod in the exp itself, but I found it interesting nonetheless / short day treatments initiated cold acclimation and freeze tolerance, which was faster in the northern ecotype compared to the southern ecotype / bud water content decreased when plants were chilled at 6 degC, no change at 0.5 degC, then rapidly increased 3 weeks later when plants were exposed to 18 degC / freezing tolerance increased from ~-10 degC to ~-20 after exposure to 6 degC, then increased further from ~-25 to ~30 when exposed to 0.5 degC, then rapidly dropped when plants were exposed to 18 degC measured through SLT_50, the temp at which 50% cellular solutes are lost / ABA concentration in the shoot apex increased n during short days
 - man17: I don't think they manipulated photoperiod
 - myking95: photoperiods changed during flushing, not chilling
 - nienstaedt66: photoperiods changed during flushing, not chilling
 - spiers74: photoperiods changed during flushing, not chilling / but I found it interesting that defoliation reduced the dormancy duration of vegetative buds that were exposed to constant chilling, but no acceleration of floral budburst
 - worrall67: _Picea abies_ / after 50 days of chilling at 4 degC, there was no difference in budburst between different photoperiod treatments / unchilled plants kept in a short photoperiod did not burst until they were exposed to 1 month of chilling / this paper's section on photoperiod is confusing because they claim that long photoperiod can induce budburst, but their results don't explicitly show that

For whichever findings we think are worthwhile exploring / including, I'll download the paper and add it and its reference to the folder where we keep all the pdfs and the Zotero export.

**Comment by lizzieinvancouver** [comment]
@ngoj1 Looks good! Can you summarize with citations added as follows:
* papers (species) x, y, z found below zero advanced or increased budburst timing or percent
* papers (species) a, b, c  found below zero delayed or decreased budburst timing or percent
* papers (species) e, f, g were inconclusive

**Comment by ngoj1** [comment]
Papers that found below zero advanced or increased budburst timing or percent:
**_Ribes nigrum_:**
 - Sønsteby, A., & Heide, O. M. (2014). Chilling requirements of contrasting black currant (Ribes nigrum L.) cultivars and the induction of secondary bud dormancy. Scientia Horticulturae, 179, 256–265. https://doi.org/10.1016/j.scienta.2014.09.038
 - Jones, H. G., Hillis, R. M., Gordon, S. L., & Brennan, R. M. (2013). An approach to the determination of winter chill requirements for different Ribes cultivars. Plant Biology, 15(s1), 18–27. https://doi.org/10.1111/j.1438-8677.2012.00590.x

Papers that found below zero delayed or decreased budburst timing or percent:
_**Picea glauca:**_
 - Man, R., & Lu, P. (2010). Effects of thermal model and base temperature on estimates of thermal time to bud break in white spruce seedlings. Canadian Journal of Forest Research, 40(9), 1815–1820. https://doi.org/10.1139/X10-129

**_Rubus idaeus_ 'Latham'**
 - Lamb, R. C. (1948). Effect of temperatures above and below freezing on the breaking of rest in the Latham raspberry. Proceedings of the American Society for Horticultural Science, 51, 313-315.

**_Malus domestica_ 'Granny Smith'**
 - Cook, N. C., Bellen, A., Cronjé, P. J. R., Wit, I. D., Keulemans, W., Van den Putte, A., & Steyn, W. (2005). Freezing temperature treatment induces bud dormancy in ‘Granny Smith’ apple shoots. Scientia Horticulturae, 106(2), 170–176. https://doi.org/10.1016/j.scienta.2005.03.009

granhus09, Picea abies just used flat 0 degC for chilling, no comparison made between different chilling temperatures.
